HW_3/forms.py

```diff
@@ -0,0 +1,15 @@
+from flask_wtf import FlaskForm
+from wtforms import StringField, PasswordField, DateField, BooleanField
+from wtforms.validators import DataRequired, Email, Length, EqualTo, Regexp
+
+
+class LoginForm(FlaskForm):
+    username = StringField('Username', validators=[DataRequired()])
+    password = PasswordField('Password', validators=[DataRequired()])
+
+
+class RegistrationForm(FlaskForm):
+    username = StringField('Username', validators=[DataRequired()])
+    email = StringField('Email', validators=[DataRequired(), Email()])
```

Comment:
Можно использовать поле `EmailField`
```suggestion
    email = EmailField('Email', validators=[DataRequired(), Email()])
```